### PR TITLE
Accelerometer

### PR DESCRIPTION
**Pull request by OcelotEmpire**
### Things Added:
- overhaul of old ThreeDRover using react-three/fiber and react-three/drei libraries
- updated 3D models to match the current rover
- greatly reduced poly count to improve performance
- 3D models rotate in real time according to a "down" vector
- shadows to make the rotation more readable
- debug tools to apply rotations manually
- support for .gltf and .glb files, allowing materials and animations
### Things Still Needed:
- RoveComm packet called AccelerometerData containing 3 floats added to manifest
- maybe some error handling to make the console less cluttered

### CODE REVIEW

File: assets/Waypoints.json
Comment (wiidler) [inline assets/Waypoints.json:1]: try not to push new waypoints, we want to keep these, so reset this file to dev

File: assets/models/tex01.png
Comment (wiidler) [inline assets/models/tex01.png:1]: no

File: src/Core/components/Accelerometer.tsx
Comment (wiidler) [inline src/Core/components/Accelerometer.tsx:1]: do we need debug panel on final merge to dev?

**Review by wiidler — CHANGES_REQUESTED** [review]
Address comments

File: src/Core/components/Accelerometer.tsx
Comment (OcelotEmpire) [reply src/Core/components/Accelerometer.tsx:1]: no, but it can stay until the hardware is installed

File: assets/models/tex01.png
Comment (OcelotEmpire) [reply assets/models/tex01.png:1]: yes

**Comment by wiidler** [comment]
Merged to https://github.com/MissouriMRDT/BaseStation_Software/pull/355

**Pull request closed without merging**